### PR TITLE
add shirt sizes information hint and optional link

### LINKED ISSUE #859: Include explanation of T-shirt types when signing up
New angels have to pick a T-shirt type. I suggest to include a help link to this page:
https://geekfeminism.fandom.com/wiki/T-shirts

Because not everyone knows what straight-cut and fitted-cut are.

### PR DESCRIPTION
**Pull request by xuwhite**
Fixes #859

### CODE REVIEW

File: resources/views/pages/registration.twig [hunk 1]
```diff
@@ -239,6 +239,14 @@
                                 'required': isTShirtSizeRequired,
                                 'required_icon': isTShirtSizeRequired,
                                 'selected': f.formData('tshirt_size', ''),
+                                'info': __('settings.profile.shirt_size.hint'),
+                                'raw_form_text': true,
+                                'form_text': (tShirtLink ? m.icon('info-circle')
+                                    ~ ' <a href="'
+                                    ~ tShirtLink|escape('html_attr')
+                                    ~ '">'
```
Comment (MyIgel) [inline resources/views/pages/registration.twig:247]: A `target="_blank" rel="noopener"` is needed here (and on the profile settings) to not discard the form content (the noopener part should not be needed nowadays but :shrug:)

File: resources/lang/en_US/default.po [hunk 2]
```diff
@@ -344,6 +344,16 @@ msgstr "To withdraw your approval, send an e-mail to <a href=\"mailto:%s\">%1$s<
...
 
+msgid "settings.profile.shirt_size.hint"
+msgstr "A straight-cut shirt has wide shoulders and a body which is almost square. "
+"A fitted-cut t-shirt has a curved side seam which comes in at the waist "
+"and goes out at the upper and lower ends. "
+"Normally a fitted-cut shirts are smaller then same size straight-cut shirts, "
+"and sizes differ between brands."
+
+msgid "settings.profile.shirt.link"
+msgstr "More information"
```
Comment (MyIgel) [inline resources/lang/en_US/default.po:355]: I would love to have some text here but :man_shrugging:

**Review by MyIgel — CHANGES_REQUESTED** [review]

File: resources/lang/en_US/default.po (hunk 2 above)
Comment (xuwhite) [reply resources/lang/en_US/default.po:355]: I'm open for suggestions

File: resources/views/pages/registration.twig (hunk 1 above)
Comment (xuwhite) [reply resources/views/pages/registration.twig:247]: done

**Pull request merged**